Formula/qmmp.rb

```diff
@@ -17,38 +16,71 @@ class Qmmp < Formula
     sha256 mojave:   "7001d36b42beb62f494c03d0e15bf5a153712ad9784af50df1e8042b52d682f6"
   end
 
-  depends_on "cmake" => :build
+  depends_on "cmake"      => :build
+  depends_on "pkg-config" => :build
+
+  # TODO: on linux: pipewire
   depends_on "faad2"
   depends_on "ffmpeg"
   depends_on "flac"
+  depends_on "game-music-emu"
+  depends_on "jack"
+  depends_on "libarchive"
   depends_on "libbs2b"
+  depends_on "libcddb"
+  depends_on "libcdio"
   depends_on "libmms"
+  depends_on "libmodplug"
   depends_on "libogg"
   depends_on "libsamplerate"
+  depends_on "libshout"
   depends_on "libsndfile"
   depends_on "libsoxr"
   depends_on "libvorbis"
+  depends_on "libxcb"
+  depends_on "libxmp"
   depends_on "mad"
   depends_on "mplayer"
   depends_on "musepack"
   depends_on "opus"
   depends_on "opusfile"
-  depends_on "qt@5"
+  depends_on "projectm"
+  depends_on "pulseaudio"
+  depends_on "qt"
   depends_on "taglib"
+  depends_on "wavpack"
+  depends_on "wildmidi"
+
+  uses_from_macos "curl"
+  uses_from_macos "libiconv"
```

Comment:
`libiconv` is a macOS-only formula as it is part of GNU glibc for Linux. So, this line is equivalent to `depends_on :macos` right now.
```suggestion
```

We probably need to add a check to make sure formula in `uses_from_macos` are not `depends_on :macos`.